model-optimizer/extensions/front/tf/CTCLossReplacement.py

```diff
@@ -134,53 +85,20 @@ def replace_sub_graph(self, graph: Graph, match: dict):
-        ctc_greedy_decoder_tf.in_port(0).get_connection().set_destination(ctc_greedy_decoder.in_port(0))
-        ctc_loss.in_port(0).disconnect()
-        transpose_tf.in_port(0).get_connection().add_destination(ctc_loss.in_port(0))
-
-        # connect logit lengths
-        ctc_greedy_decoder_tf.in_port(1).disconnect()
-        ctc_loss.in_port(1).connect(reduce_to_seq_len_node.out_port(0))
-
-        # connect labels to ctc_loss
-        squeeze_op = create_op_with_const_inputs(graph, Squeeze, {1: int64_array([2, 3])})
-        cast_labels_op = Cast(graph, {'name': output_sparse_to_dense_name + '/CastLabels', 'dst_type': np.int32}).create_node()
-        squeeze_op.in_port(0).connect(ctc_greedy_decoder.out_port(0))
-        cast_labels_op.in_port(0).connect(squeeze_op.out_port(0))
-        ctc_loss.in_port(2).connect(cast_labels_op.out_port(0))
-
-        # connect label lengths
-        equal_op = create_op_with_const_inputs(graph, Equal, {1: np.array([-1], dtype=np.int32)},
-                                               {'name': output_sparse_to_dense_name + '/Equal'})
-        equal_op.in_port(0).connect(cast_labels_op.out_port(0))
-        labels_shape_op = Shape(graph, {'name': output_sparse_to_dense_name + '/ShapeOf'}).create_node()
-        labels_shape_op.in_port(0).connect(equal_op.out_port(0))
-        broadcast_one = create_op_with_const_inputs(graph, Broadcast, {0: np.array([1], dtype=np.int32)},
-                                                    {'mode': 'numpy',
-                                                     'name': output_sparse_to_dense_name + '/One'})
-        broadcast_one.in_port(1).connect(labels_shape_op.out_port(0))
-        broadcast_zero = create_op_with_const_inputs(graph, Broadcast, {0: np.array([0], dtype=np.int32)},
-                                                     {'mode': 'numpy',
-                                                      'name': output_sparse_to_dense_name + '/Zero'})
-        broadcast_zero.in_port(1).connect(labels_shape_op.out_port(0))
-
-        select_node = Select(graph, {'name': output_sparse_to_dense_name + '/Select'}).create_node()
-        select_node.in_port(0).connect(equal_op.out_port(0))
-        select_node.in_port(1).connect(broadcast_zero.out_port(0))
-        select_node.in_port(2).connect(broadcast_one.out_port(0))
-        label_length_node = create_op_with_const_inputs(graph, ReduceSum, {1: int64_array([1])},
-                                                      op_attrs={'name': output_sparse_to_dense_name + '/LabelLength',
-                                                                'keep_dims': False})
-        label_length_node.in_port(0).connect(select_node.out_port(0))
-        ctc_loss.in_port(3).connect(label_length_node.out_port(0))
-
-        # set source for output of new sub-graph and remove old nodes
+                                   'unique': unique,
+                                   'logits_time_major': logits_time_major}).create_node()
```

Comment:
we don't have `logits_time_major` attribute for CTCLoss in our opset. You should use it here to decide whether to insert `Transpose` or not for input data